.github/workflows/unit_tests.yml

```diff
@@ -115,38 +115,26 @@ jobs:
     #
     # 6 checks:
     # for each OS (ubuntu, macos, windows):
```

Comment:
```suggestion
    # for each OS (ubuntu, macos, macos-13, windows):
    # (NOTE: macos-13=x86_64, macos>13=arm64)
```

Reply:
actually, the `    # 6 checks:` above needs to be changed to "8 checks" 🤔 

and it'd be nice to have a changelog entry saying that we now require Python 3.10 minimum. Could you add that please @larsoner ? If not, let me know and I'll do it at some later point. Thanks again.